src/XeroPHP/Remote/Query.php

```diff
@@ -20,6 +25,16 @@ class Query {
     private $page;
     private $offset;
 
+    /**
+     * @var bool|null
+     */
+    private $includeArchived = null;
```

Comment:
@calcinai we set it to false and not add it to query if it is false, because by default includeArchived is not shown anyway.